### PR TITLE
Add keywords to logs

### PR DESCRIPTION
**Pull request by libertylocked**
`keywords` is a `string` that gets emitted in `LogRecordAdded` event. Database indexing services that listen for contract events can store the keywords along with the hash of a file, allowing record searching by keywords.

### CODE REVIEW

File: contracts/LinniaRecords.sol [hunk 1]
```diff
@@ -14,15 +14,17 @@ contract LinniaRecords is Ownable {
         uint sigCount;
         mapping (address => bool) sigs;
         uint irisScore;
-        // For now the record types are
-        // 0 nil, 1 Blood Pressure, 2 A1C, 3 HDL, 4 Triglycerides, 5 Weight
         uint recordType;
```
Comment (godfreyhobbs) [inline contracts/LinniaRecords.sol]: consider doing one of the following:
1. removing recordType
2. Clearly defining why recordType is necessary

File: contracts/LinniaRecords.sol [hunk 2]
```diff
@@ -72,31 +76,38 @@ contract LinniaRecords is Ownable {
     /// Add a record by user without any provider's signatures.
     /// @param fileHash the hash of the original unencrypted file
     /// @param recordType the type of the record
+    /// @param keywords keywords for the record
```
Comment (godfreyhobbs) [inline contracts/LinniaRecords.sol]: consider improving the comment.  Maybe mentioning something like the following:
`this use used at different software layers for indexing`

Comment (godfreyhobbs) [reply contracts/LinniaRecords.sol]: Also, can keywords be empty?

**Comment by buendiadas** [comment]
LGTM overall

Open question, no PR blocker wouldn't it be interesting to have keywords indexed as well? So I could look for a type of data directly without making any intermediate DB.

**Comment by libertylocked** [comment]
@buendiadas posibly, but keywords right now aren't editable after adding. also the keywords aren't attested.

**Comment by buendiadas** [comment]
@libertylocked I don't think it can be done with string, sth like `bytes`. 

Even something like a namehash could be interesting (could enable meta inheritance, and a bit extra privacy)

Food for thoughts!

**Pull request merged**